dist/stat_data.py

```diff
@@ -352,6 +352,44 @@ def __init__(self, name, desc, flags=''):
     CapacityStat('fsync_all_fh_total', 'background fsync file handles considered'),
     CapacityStat('fsync_all_time', 'background fsync time (msecs)', 'no_clear,no_scale'),
 
+    ##########################################
+    # Checkpoint statistics
+    ##########################################
+    CheckpointStat('checkpoint_custom_dsrc', 'number of custom data sources checkpointed'),
+    CheckpointStat('checkpoint_fsync_post', 'fsync calls after allocating the transaction ID'),
+    CheckpointStat('checkpoint_fsync_post_duration', 'fsync duration after allocating the transaction ID (usecs)', 'no_clear,no_scale'),
+    CheckpointStat('checkpoint_generation', 'generation', 'no_clear,no_scale'),
+    CheckpointStat('checkpoint_handle_applied', 'most recent handles applied'),
+    CheckpointStat('checkpoint_handle_duration', 'most recent duration for gathering all handles (usecs)', 'no_clear,no_scale'),
+    CheckpointStat('checkpoint_handle_duration_apply', 'most recent duration for gathering applied handles (usecs)', 'no_clear,no_scale'),
+    CheckpointStat('checkpoint_handle_duration_skip', 'most recent duration for gathering skipped handles (usecs)', 'no_clear,no_scale'),
+    CheckpointStat('checkpoint_handle_skipped', 'most recent handles skipped'),
+    CheckpointStat('checkpoint_handle_walked', 'most recent handles walked'),
+    CheckpointStat('checkpoint_hs_pages_reconciled', 'number of history store pages caused to be reconciled'),
+    CheckpointStat('checkpoint_pages_reconciled', 'number of pages caused to be reconciled'),
+    CheckpointStat('checkpoint_pages_visited_internal', 'number of internal pages visited'),
+    CheckpointStat('checkpoint_pages_visited_leaf', 'number of leaf pages visited'),
+    CheckpointStat('checkpoint_prep_max', 'prepare max time (msecs)', 'no_clear,no_scale'),
```

Comment:
These were moved from TxnStat. But also all of the prefix phrases were moved. That is a problem in T2. Things are alphabetized and so all of these now will not sort or be shown together. When they had `transaction checkpoint` as a prefix, then they sorted together. And seeing just `generation` (further above) is going to be confusing as there are several generation numbers.

Reply:
These being a `CheckpointStat` means that `checkpoint` automatically gets prepended to all of the descriptions, so the only difference here is that there is no longer a superfluous "transaction transaction" string at the start of the description. I'll upload some screenshots in another comment.

Reply:
Before:

![Screenshot 2023-08-18 at 4 18 01 pm (2)](https://github.com/wiredtiger/wiredtiger/assets/1955860/a11c4371-0a50-4729-b142-7f50ffff7831)

After:

![Screenshot 2023-08-18 at 4 16 07 pm (2)](https://github.com/wiredtiger/wiredtiger/assets/1955860/27346335-e267-4145-af75-dbbe42edee41)